stressor/package.json

```diff
@@ -4,14 +4,14 @@
   "description": "Stress test Test Plans via CI workflows",
   "main": "stress-test.mts",
   "scripts": {
-    "stress-test": "node --loader ts-node/esm stress-test.mts",
     "cancel-runs": "node --loader ts-node/esm cancel-all-workflows.mts"
   },
   "author": "",
   "license": "MIT",
   "dependencies": {
```

Comment:
should we add `tsx` to the list of dependencies here? i had to install it separately when i went to run the command via the instructions above.

Reply:
Sounds good!

(This shouldn't be technically necessary because [`npx` is supposed to fetch the requested patch for you](https://docs.npmjs.com/cli/v7/commands/npx). I'm happy to help diagnose if that's something you want to look in to. Regardless, it seems wise to include the dependency as you've requested to avoid version drift.)

Reply:
oh yes, `npx` prompted me before it fetched `tsx`. i should have said this was not a big deal :)